apps/auction2/client.js

```diff
@@ -1,10 +1,25 @@
 import Auction from '../../models/auction.coffee'
 import CurrentUser from '../../models/current_user.coffee'
+import Artist from '../../models/artist.coffee'
 import ClockView from '../../components/clock/view.coffee'
 import { data as sd } from 'sharify'
 import _ from 'underscore'
 import MyActiveBids from '../../components/my_active_bids/view.coffee'
+import Backbone from 'backbone'
 
+import Params from '../../components/commercial_filter/models/params.coffee'
+import Filter from '../../components/commercial_filter/models/filter.coffee'
+import TotalView from '../../components/commercial_filter/views/total/total_view.coffee'
+import SortView from '../../components/commercial_filter/views/sort/sort_view.coffee'
+import PriceFilterView from '../../components/commercial_filter/filters/price/price_filter_view.coffee'
+import MediumFilterView from '../../components/commercial_filter/filters/medium/medium_filter_view.coffee'
+// import GeneIdsFilterView from '../../components/commercial_filter/filters/gene_ids/gene_ids_filter_view.coffee'
```

Comment:
This is another filter view that I added, which is intended to match the spec. Basically a checkbox where you can select one or multiple genes to filter by.

Currently if you pass multiple `gene_ids` into the filter api, it returns artworks that have _all_ of the passed-in genes, not artworks that have one or the other (`and` vs `or`). I kept it here for reference, but it'll require an additional API change to actually be useful.